.experimenter.yaml

```diff
@@ -15,6 +15,9 @@ homescreen:
   hasExposure: true
   exposureDescription: ""
   variables:
+    onboarding:
```

Comment:
Would it make sense to have a new `variable` for each of the homescreen onboarding features? 
Or a collection similar to the below `sections-enabled` which would allow individually controlling each onboarding feature shown on homescreen?

Reply:
> Or a collection similar to the below sections-enabled which would allow individually controlling each onboarding feature shown on homescreen?

Agree, this approach will more scalable!